client/web/src/components/views/device-details-view.tsx

```diff
@@ -101,20 +116,46 @@ export default function DeviceDetailsView({
             <tbody>
               <tr>
                 <td>Tailscale IPv4</td>
-                <td>{node.IP}</td>
+                <td>
+                  <QuickCopy
+                    primaryActionValue={node.IPv4}
+                    primaryActionSubject="IPv4 address"
+                  >
+                    {node.IPv4}
+                  </QuickCopy>
+                </td>
               </tr>
               <tr>
                 <td>Tailscale IPv6</td>
-                <td>{node.IPv6}</td>
+                <td>
+                  <QuickCopy
+                    primaryActionValue={node.IPv6}
+                    primaryActionSubject="IPv6 address"
+                  >
+                    {node.IPv6}
```

Comment:
We should probably use NiceIP here as well.  Currently, on small screens this truncates on the right instead of the middle.